modules/serverless-ossm-v2x-jwt.adoc

```diff
@@ -47,13 +47,13 @@ spec:
 ----
 <1> The path on your application to collect metrics by system pod.
 <2> The path on your application to probe by system pod.
```

Comment:
The [doc](https://docs.openshift.com/container-platform/4.7/serverless/networking/serverless-ossm-jwt.html#serverless-ossm-v2x-jwt_serverless-ossm-jwt) already explains you need to add `/metrics` and `/healthz` as they are accessed by system pod.

I think you assumed that the `probe` means liveness/readiness probe but here it means probe from system pod.

Reply:
This is totally unclear. Docs says here:

> _The path on your application to probe by system pod._

My application doesn't have a `/healthz` endpoint at all. How do I need to know what paths should I configure here.

Maybe this should be written as?

> _This path list needs to cover all readiness and liveness probes of your applications in this namespace, plus `/metrics` and `/healthz` paths regardless if they are defined by your applications_

Reply:
Maybe we should also add why those paths are required, I mean what system component (knative/kourier/istio) requires the `/healthz` path to be accessible.

Reply:
`/healthz` is required as Knative sets up a probe against the queue-proxy as well, which is done against a `/healthz` endpoint. So technically, it's the stamped out Deployment requiring that. This has nothing to do with the networking plugins inherently.

Reply:
:bow: @markusthoemmes. I think such explanation needs to be added here. It needs to be clear that users needs to collect all health endpoints of their apps and add those that are users by proxy components.

Reply:
@cardil please add another Jira for this since this isn't part of the scope for the mTLS stuff we're trying to add at the minute, we can't keep adding issues to this PR or it's never going to be finished